### DISCUSSION TITLE
bbox and geometry show incorrect information when using 2023-11-14-alpha.0 release version

### DISCUSSION THREAD

**Discussion by shuu-yoshida** [OvertureMaps/data#93] [Discussion]
I am using the theme=places/type=place dataset with addresses.country filtered to 'JP', and I am reporting that the location information in bbbox and geometry is clearly not showing the location of Japan.
I have not checked what is happening in other countries.
Also, previous versions did not have the problem.

Attached is a sample data with addresses.country filtered by 'JP'.

We would appreciate it if you could check it.
Thank you in advance.

[export.csv](https://github.com/OvertureMaps/data/files/13387286/export.csv)

**Comment by shuu-yoshida** [comment]
A detailed check of the data revealed that there were only a few hundred to a thousand locations with large discrepancies in location information, which is a small portion of the total data.
We will try to filter out the data that is clearly out of range.

**Comment by D3M05** [comment]
I haven't looked at previous releases, but a quick glance at the November data shows about 5k records with non matching location information in the US.  Source did not matter, and the facebook page didn't show the incorrect data.  Some had correct street and geometry, but wrong city/country.  Others had correct address, but wrong geometry.  

Here is a quick query to see an example of incorrect US records.

```
WHERE (addresses).country = 'US' AND NOT (
    -- Continental United States
    (latitude BETWEEN 24.396308 AND 49.384358 AND longitude BETWEEN -125.001651 AND -66.93457)
    OR
    -- Hawaii
    (latitude BETWEEN 18.910361 AND 22.23559 AND longitude BETWEEN -160.247482 AND -154.806773)
    OR
    -- Puerto Rico
    (latitude BETWEEN 17.926869 AND 18.515683 AND longitude BETWEEN -67.271492 AND -65.589752)
    OR
    -- Alaska
    (latitude BETWEEN 51.209708 AND 71.5388001 AND longitude BETWEEN -179.148909 AND -129.98047)
)

```

**Comment by shuu-yoshida** [comment]
@d3mon187 

Thank you for your comment.
It looks like I can get the data I need by filtering latitude and longitude as in the query you presented.
I will try to use that method for the current situation in anticipation of future updates.
